ansible/roles/images/tasks/load.yaml

```diff
@@ -9,12 +9,28 @@
   register: images_cache_find_result
   changed_when: false
 
-- name: import images from files in the local images cache  # noqa no-changed-when
-  shell:
-    executable: /bin/bash
-    cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+- block:
+    - name: import images from files in the local images cache # noqa no-changed-when
+      shell:
+        executable: /bin/bash
+        cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+      when:
+        - item | regexp(^1\.(\d)+\.(\d)+_images\.tar\.gz$)
+        - item.split('.')[1] |int < 24
```

Comment:
@faiq @supershal @dlipovetsky  what do you think about taking this opportunity and renaming these k8s >=v1.24 image files?

Currently its:
```
➜  ~ aws s3 ls s3://downloads.mesosphere.io/dkp/airgapped/kubernetes-images/
2022-04-04 12:58:39  205642603 1.21.6_images.tar.gz
2022-04-04 13:00:22  208045884 1.21.6_images_fips.tar.gz
```

Maybe we can start creating any packages for >1.24 
```
kubernetes-images-1.21.6-d2iq.1.tar.gz
kubernetes-images-1.21.6-d2iq.1_fips.tar.gz
```

This `when` condition would be much simpler to write.
This would also aligns it better with:
```
➜  ~ aws s3 ls s3://packages.d2iq.com/dkp/containerd/
2022-05-25 16:46:22          0
2022-06-01 09:51:40   50027539 containerd-1.4.13-d2iq.1-centos-7.9-x86_64.tar.gz
2022-06-01 09:51:56   54357021 containerd-1.4.13-d2iq.1-centos-7.9-x86_64_fips.tar.gz
```

Reply:
This also has the benefit that it will not break any of our e2e automation. As we can still have both image tars for 1.24.4

Reply:
I like the idea of a name change

Reply:
+1

And we can take this opportunity to change `_fips` to `-fips` :partying_face:

Reply:
+1

Reply:
names are now -fips for images. 

good luck to whoever is changing containerd and os-packages